### PR TITLE
Jamesclark

### PR DESCRIPTION
**Pull request by jamesjclark1**
jamesjclark1 Control Flow complete

### CODE REVIEW

File: ControlFlow/Program.cs [hunk 1]
```diff
@@ -15,22 +16,36 @@ public static class ControlFlow
...
         // they are equal. Assume non-negative values.
         public static string LandscapeOrPortrait(int width, int height)
         {
-            return default;
+            if (width < height)
+                return "Portrait";
+            if (width > height)
+                return "Landscape";
+            else
+                return "Square";
```
Comment (elijahsteve) [inline ControlFlow/Program.cs:48]: I would recommend doing a singular variable that changes between the different desired strings, then just simply return that variable instead of the strings.

```
var result = "Portrait"
result = "Landscape"
result = "Square"

return result;
```

Something to this effect

File: ControlFlow/Program.cs [hunk 2]
```diff
@@ -48,7 +63,23 @@ public static string LandscapeOrPortrait(int width, int height)
         // Suspended" instead. Don't Worry about input validation.
         public static string SpeedTrap(int speedLimit, int speedCar)
         {
-            return default;
+            var SD = speedCar - speedLimit;
+            var DS = 0;
```
Comment (elijahsteve) [inline ControlFlow/Program.cs:67]: I would always recommend naming your variables in plain English or in Pascal case (pascalCase).
This way when other programmers are reviewing your code they can make sense of the variables and their purpose a lot faster.

**Review by elijahsteve — COMMENTED** [review]
All in all good job, some minor stuff with returns that can be simplified to improve code readability.

File: ControlFlow/Program.cs (hunk 1 above)
Comment (jamesjclark1) [reply ControlFlow/Program.cs:48]: Thank you, when I did this exercise I didn't know about var result yet.

File: ControlFlow/Program.cs [hunk 3]
```diff
@@ -48,7 +63,23 @@ public static string LandscapeOrPortrait(int width, int height)
...
+            if (speedLimit >= speedCar)
+                return Convert.ToString("Okay");
+            else
+            {
+                for (int TF = 5;
+                    TF <= SD;
+                    TF += 1)
```
Comment (elijahsteve) [inline ControlFlow/Program.cs:74]: This for loop should be all on one line, the way it is currently is very confusing to read.

File: ControlFlow/Program.cs [hunk 4]
```diff
@@ -48,7 +63,23 @@ public static string LandscapeOrPortrait(int width, int height)
...
+                    if (TF % 5 == 0)
+                        DS++;
+                if (DS > 12)
+                    return Convert.ToString("License Suspended");
```
Comment (elijahsteve) [inline ControlFlow/Program.cs:78]: Do not need to convert "License Suspended" into a string because it is already a string(anything in between "quotation marks" is read as a string, even numbers).
